x/genutil/expected.go

```diff
@@ -0,0 +1,32 @@
+package genutil
```

Comment:
Having an "expected" package is great but it doesn't help if the main module imports the other modules directly, right?

Reply:
good point, that totally slipped past me, there needs to be further abstraction to remove references to `staking` (which is referenced to abtain the MsgCreateValidator type and default bond denom right now) 

and also `genaccounts` (which is only used by testnet, which I originally had in a separate module, then reluctantly moved back in ignoring this)

Reply:
ref https://github.com/cosmos/cosmos-sdk/pull/4159#discussion_r282713346 I think greater abstraction should come soon but in new PRs